netdata-installer.sh

```diff
@@ -90,11 +92,11 @@ defer_error_highlighted() {
   NETDATA_DEFERRED_ERRORS="${TPUT_YELLOW}${TPUT_BOLD}${NETDATA_DEFERRED_ERRORS}\n* ${1}${TPUT_RESET}"
 }
 
+# shellcheck disable=SC2059,SC2086
```

Comment:
SC2086 is not an issue in this function, and SC2059 should just be fixed instead of being suppressed.